themes/HeroesLounge-Theme/pages/home.htm

```diff
@@ -1,55 +1,75 @@
-title = "Home"
-url = "/"
-layout = "clean"
-is_hidden = 0
-contentType = "html"
-force_show = 0
-
-[UpcomingMatches]
-daysInFuture = 14
-showLogo = 1

```

Comment:
Is this configured somewhere else now?

Reply:
This is now configured inside of the `user/matches` partial, which already existed but wasn't used for some reason